### PR TITLE
Forms, Whiteboard, Planner, iOS 14, OLE hardening

### PR DESCRIPTION
**Pull request by noreilly-ccx**
This release incorporates changes based on the latest Microsoft IRAP assessment, ACSC OLE guidance, and iOS updates.

Overall
- Updated System Center Configuration Manager updated to Microsoft Endpoint Configuration Manager
- Updated Intune to Microsoft Endpoint Manager - Intune to reflect the move into the Microsoft Endpoint Manager suite.

Blueprint Modifications 
- Platform
        - Updated to include Whiteboard, Planner, and Forms within the Office 365 scope
        - Updated MCAS policy to include information on monitoring and detection of access from anonymous IP addresses
        - Updated the log analytics section to address what logs are to be configured to be sent to the instance
        - Updated the list of Office 365 roles available in PIM
- Office 365
        - Updated to include Whiteboard, Planner and Forms
        - Updated roles based access control to account for additional Office 365 roles
        - Updated the Unified Audit log for 10 year retention
        - Updated the Office 365 Groups terminology to Microsoft 365 groups
- Client Devices
        - Updated for OLE hardening
        - Updated for iOS 14.0 uplift

ABAC Modifications
- Hybrid - Intune configuration for Windows devices
         - Updated to include a script to block OLE extensions
- Hybrid - platform
         - Updated to include the Microsoft Endpoint Manager and Azure Active Directory Diagnostic log configuration
         - Updated the Office 365 Groups terminology to Microsoft 365 groups
- Hybrid - office 365
         - Updated the organisational settings for the Whiteboard, Planner, and Forms add-ins
         - Updated the Audit logging to include a Audit Retention Policy
- platform
         - Updated the Office 365 Groups terminology to Microsoft 365 groups
- office 365
         - Added in organisational settings
         - Updated the Audit logging to include a Audit Retention Policy
- Intune configuration
         - Updated to include a script to block OLE extensions
         - Updated to include a WIFI configuration
         - Updated to include a per app vpn configuration 
         - Updated to iOS configuration
- Intune compliance
         - Updated to reference iOS 14
- Hybrid - Intune compliance for iOS devices
         - Updated to reference iOS 14
- Intune applications
         - Updated to include the whiteboard and planner apps
- Intune software updates
         - Updated to include a iOS update policy
- Hybrid - Intune applications for iOS devices
         - Updated to include the whiteboard and planner apps
- Hybrid - Intune configuration for iOS devices
         - Updated to include a script to block OLE extensions
         - Updated to include a WIFI configuration
         - Updated to include a per app vpn configuration 
         - Updated to iOS configuration

### CODE REVIEW

File: blueprint/office-365.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ Power Platform | The Power Platform includes PowerApps, Power BI, and Microsoft
 Security and Compliance | Office 365 provides Security and Compliance tools which can be utilised to implement an organisation's Information Management Policy and to assist with information governance.
 Exchange Online Protection | Exchange Online protection is a cloud hosted email security service (Mail Gateway) that acts to filter spam and scan for viruses on email entering and leaving Exchange Online.
 Office 365 Advanced Threat Protection (ATP) | Office 365 ATP is a cloud-based mail threat protection service. The service provides protection against unknown malware and viruses through the use of robust zero-day protection and inclusion of features to safeguard an organisation from harmful links in real time.
+Microsoft Whiteboard | Microsoft Whiteboard is a digital whiteboard that facilitates collarboration between users. It has integrations with the Office 365 suite and physical endpoints such as the Microsoft Surface hub.
```
Comment (zedzedzed) [inline blueprint/office-365.md:20]: Suggest running this file through a spell checker. There's a couple, eg collarboration

File: blueprint/office-365.md [hunk 2]
```diff
@@ -1627,3 +1615,53 @@ Add trust senders and domains | Not configured | When users interact with domain
...
+Easy sharing of Whiteboard from Surface hub | Agency Decision | Depending on the location and configuration of the Surface Hub/s within the organisation this setting may be configured.   
+
+## Microsoft Forms
+Microsoft Forms is a form and survey tool. It offers a variety of external sharing and collaboration options including:
+* Sharing links - Sharing the link to external party for form population.
+* Collaboration of layout and structure - Collaboration on the design of a form.
+* Sharing templates - Sharing a form as a template for external parties to leverage.
+* Sharing results - Sharing the results of a form with external parties.
+
+By default, Microsoft Forms can be configured to capture the names of all people within the organisation who complete a form or survey. This can also be configured on a form by form basis. Forms can also be configured to allow images from Bing and videos from YouTube to be embedded.
```
Comment (zedzedzed) [inline blueprint/office-365.md:1648]: What is the default? Maybe remove "By default" or add the default setting.

File: blueprint/office-365.md [hunk 3]
```diff
@@ -1627,3 +1615,53 @@ Add trust senders and domains | Not configured | When users interact with domain
...
+## Microsoft Forms
+Microsoft Forms is a form and survey tool. It offers a variety of external sharing and collaboration options including:
+* Sharing links - Sharing the link to external party for form population.
+* Collaboration of layout and structure - Collaboration on the design of a form.
+* Sharing templates - Sharing a form as a template for external parties to leverage.
+* Sharing results - Sharing the results of a form with external parties.
+
+By default, Microsoft Forms can be configured to capture the names of all people within the organisation who complete a form or survey. This can also be configured on a form by form basis. Forms can also be configured to allow images from Bing and videos from YouTube to be embedded. 
+
+Surveys and forms can be leveraged for potentially malicious purposes. Phishing protection for Microsoft Forms scans forms that only allow in organisation responses for common phishing questions. If they are detect the form is prevented from being distributed or shared.
```
Comment (zedzedzed) [inline blueprint/office-365.md:1650]: I think this paragraph needs some rewording.

File: blueprint/office-365.md [hunk 4]
```diff
@@ -1627,3 +1615,53 @@ Add trust senders and domains | Not configured | When users interact with domain
 Spoofing filter settings | Enable antispoofing protection: On | Allows the Agency to filter email from senders who are spoofing domains.
 Enable Unauthenticated Sender Feature | Enable Unauthenticated Sender: On | Displays a notification to users in Outlook when a sender fails authentication checks.
 Actions | If email is sent by someone who's not allowed to spoof your domain: Quarantine the message | Specify an action in an event an unauthorised user spoofs a domain.
+
+## Microsoft Whiteboard
+Microsoft Whiteboard is a hosted whiteboarding tool used for collaboration. It integrates with Office 365 services such as Teams and functions on devices such as the Microsoft Surface hub, Windows and iOS. Acess to use the tool can be controlled globally.
+
+Microsoft leveages diagnostic data to both address issues and develop updates to the Microsoft Whiteboard tool. The level of data provided to Microsoft from clients utilising the Whiteboard application can be controlled. 
+
+Diagnostic data level options are:
```
Comment (zedzedzed) [inline blueprint/office-365.md:1624]: Join these paragraphs, they are related

Typo: leveages

Suggest simplifying:

`Microsoft uses diagnostic data to address issues and continually improve Microsoft Whiteboard...`

File: blueprint/office-365.md [hunk 5]
```diff
@@ -1627,3 +1615,53 @@ Add trust senders and domains | Not configured | When users interact with domain
...
+
+* Required – The minimum amount of data necessary to keep Whiteboard secure, up-to-date, and performing as expected on devices.
+* Optional – Additional data to help make product improvements and provides enhanced information to help detect, diagnose, and fix issues. All data under the required setting is also included.
+* Neither - No diagnostic data about the Whiteboard client software running on the devices is sent to Microsoft. 
+
+The Whiteboard tool also offers optional connected experiences. These experiencies are offered directly to users by Microsoft and include items such as Bing image search, Word or PowerPoint document insertion, and Ink beatufiication. When the connected services are leveraged, the required service data is provided Microsoft. This data is separate from any diagnostic data provided.
```
Comment (zedzedzed) [inline blueprint/office-365.md:1630]: Typo: beatufiication

service data is provided `to` Microsoft.

File: blueprint/office-365.md [hunk 6]
```diff
@@ -1627,3 +1615,53 @@ Add trust senders and domains | Not configured | When users interact with domain
...
+
+The Whiteboard tool can be used with the Microsoft Suface hub appliance. When using the tool easy sharing can be configured. Easy sharing allows sharing of Whiteboard sessions without logging into the Surface Hub appliance.
```
Comment (zedzedzed) [inline blueprint/office-365.md:1632]: typo: Suface

File: blueprint/office-365.md [hunk 7]
```diff
@@ -1627,3 +1615,53 @@ Add trust senders and domains | Not configured | When users interact with domain
...
+Record names by default | Disabled | Prevents the recording of user names where not required. 
+Bing search and YouTube video embedding | Disabled | Prevents the use of Bing images and Youtube images within Forms. 
+Phishing protection | Enabled | Blocks forms from being shared or distributed if they contain common phishing questions.
+
+## Microsoft Planner
+Microsoft Planner is a task planning and assignment tool which can intergrate with other Office 365 services such as Microsoft Teams. Plans developed within Planner can be published into calenders through an iCalendar feed.
+
+Decision Point | Design Decision | Justification
+--- | --- | ---
+Publishing through iCalendar feed | Enabled | Internet calendars will be enabled, it is up to the Agency which calendars they wish to publish.
```
Comment (zedzedzed) [inline blueprint/office-365.md:1667]: Let's chat what this means

**Review by zedzedzed — CHANGES_REQUESTED** [review]

File: blueprint/client-devices.md [hunk 8]
```diff
@@ -147,31 +147,31 @@ Deployment Design Decisions for hybrid implementations
...
+Cloud native | MECM is not possible with cloud native | Intune | No on-premises infrastructure required for management. Well suited for Azure AD joined workstations for a full cloud solution.
+Hybrid with Intune management | Suitable for:<br> * No MECM or<br>* MECM with co-management enabled and Workloads set to Intune | Intune | Existing on-premises MECM infrastructure with Hybrid Azure AD joined workstation. Well suited for an agency that does not have MECM or does not want to use MECM to manage workstations.
+Hybrid with MECM management | MECM with co-management enabled and Workloads set to MECM | MECM | Existing on-premises MECM infrastructure with Hybrid Azure AD joined workstations. Well suited for an agency that has a significant investment in MECM and requires a more gradual migration to Intune. Individual Workloads can be targeted and pilot-tested in Intune.
 
-The following image displays an overview diagram of SCCM Co-Management.
+The following image displays an overview diagram of MECM Co-Management.
 
-![SCCM Co-Management diagram](/assets/images/cd-sccm-comgmt.png)
+![MECM Co-Management diagram](/assets/images/cd-SCCM-comgmt.png)
```
Comment (zedzedzed) [inline blueprint/client-devices.md:174]: This may produce a 404 if not lowercased: `cd-sccm-comgmt.png`

File: blueprint/client-devices.md [hunk 9]
```diff
@@ -659,7 +659,7 @@ It is vital to have a robust and reliable patch management solution based on ind
 For Microsoft Windows environments the primary patching technologies are:
 
 * Windows Server Update Service – WSUS enables administrators to deploy the most recent Microsoft updates. A WSUS server connects directly to Microsoft Update or an "upstream" WSUS server. This allows administrators to control what updates are applied and when, rather than having every computer on the network going to the Internet and installing every available update immediately
-* Microsoft System Centre Configuration Manager (SCCM) –SCCM integrates with a WSUS server to deliver patch management. WSUS obtains updates from the internet and SCCM is used to approve and deploy the updates. Using SCCM to deploy software updates allows for more control over many aspects of the process such as targeting, maintenance windows, scheduling, and reporting
```
Comment (zedzedzed) [inline blueprint/client-devices.md:662]: I think the product name is in US spelling: `System Center`

File: blueprint/client-devices.md [hunk 10]
```diff
@@ -1198,20 +1215,20 @@ Securing iOS devices Design Decisions for all agencies and implementation types.
...
+Device passcode | Device passcode of 14 characters or above. Alphanumic in nature and must contain a minimum of 1 special character | iOS devices, by default, is encrypted once a passcode is provided to the device. Configured in line with ISM requirements on password length.
 Biometrics | Disable | This is in line with the ACSC iOS Secure Configuration Hardening guide for PROTECTED devices.
 Mobile Device Management | Enable | Mobile Device Management provides the Agency better auditing tools on the device. In line with ACSC iOS Secure Configuration Hardening guide for PROTECTED devices
 Maximum Auto-Lock  | 2 minutes | Auto Lock will lock a device if it is inactive for specified time
-Virtual Private Network (VPN)  | Not Configured | VPN will not be setup as part of the iOS device. Per-app VPN will be set up to secure communication between the device and the Agencies data
+Virtual Private Network (VPN)  | Configured | Per-app VPN will be set up to secure communication between the device and the Agencies data. This is in line with ACSC iOS Secure Configuration Hardening guide
```
Comment (zedzedzed) [inline blueprint/client-devices.md:1231]: For clarity, has this become a must in this iteration of the iOS hardening guidance?

File: blueprint/office-365.md [hunk 11]
```diff
@@ -17,6 +17,9 @@ Power Platform | The Power Platform includes PowerApps, Power BI, and Microsoft
 Security and Compliance | Office 365 provides Security and Compliance tools which can be utilised to implement an organisation's Information Management Policy and to assist with information governance.
 Exchange Online Protection | Exchange Online protection is a cloud hosted email security service (Mail Gateway) that acts to filter spam and scan for viruses on email entering and leaving Exchange Online.
 Office 365 Advanced Threat Protection (ATP) | Office 365 ATP is a cloud-based mail threat protection service. The service provides protection against unknown malware and viruses through the use of robust zero-day protection and inclusion of features to safeguard an organisation from harmful links in real time.
+Microsoft Whiteboard | Microsoft Whiteboard is a digital whiteboard that facilitates collaboration between users. It has integrations with the Office 365 suite and physical endpoints such as the Microsoft Surface hub.
+Microsoft Forms | Microsoft Forms is a online survey and form tool.
```
Comment (zedzedzed) [inline blueprint/office-365.md:21]: `is an online`

File: blueprint/office-365.md [hunk 12]
```diff
@@ -1627,3 +1615,51 @@ Add trust senders and domains | Not configured | When users interact with domain
...
+* Optional – Additional data to help make product improvements and provides enhanced information to help detect, diagnose, and fix issues. All data under the required setting is also included.
+* Neither - No diagnostic data about the Whiteboard client software running on the devices is sent to Microsoft. 
+
+The Whiteboard tool also offers optional connected experiences. These experiences are offered directly to users by Microsoft and include items such as Bing image search, Word or PowerPoint document insertion, and Ink beautification. When the connected services are leveraged, the required service data is provided to Microsoft. This data is separate from any diagnostic data provided.
+
+The Whiteboard tool can be used with the Microsoft Surface hub appliance. When using the tool easy sharing can be configured. Easy sharing allows sharing of Whiteboard sessions without logging into the Surface Hub appliance. 
+
+Decision Point | Design Decision | Justification
+--- | --- | ---
+Whiteboard for everyone in the organisation | Enabled | Allowing the use if Whiteboard within the organisation.
```
Comment (zedzedzed) [inline blueprint/office-365.md:1633]: Swap `if` with `of`

File: blueprint/office-365.md [hunk 13]
```diff
@@ -1627,3 +1615,51 @@ Add trust senders and domains | Not configured | When users interact with domain
...
+Microsoft Forms is a form and survey tool. It offers a variety of external sharing and collaboration options including:
+* Sharing links - Sharing the link to external party for form population.
+* Collaboration of layout and structure - Collaboration on the design of a form.
+* Sharing templates - Sharing a form as a template for external parties to leverage.
+* Sharing results - Sharing the results of a form with external parties.
+It can also be configured to allow images from Bing and videos from YouTube to be embedded.
+
+Microsoft Forms is capable of capturing the names of the organisational users who complete a form or survey. This functionality can also be configured globally or on a form by form basis.  
+
+Surveys and forms can be leveraged for potentially malicious purposes. Phishing protection for Microsoft Forms scans forms for common phishing questions. If they are detect the form is prevented from being distributed or shared. Phishing protection only protects internal organisation forms.
```
Comment (zedzedzed) [inline blueprint/office-365.md:1648]: `If they are detected, the form`

File: blueprint/client-devices.md (hunk 9 above)
Comment (noreilly-ccx) [reply blueprint/client-devices.md:662]: This issue is not in my code - it has been updated to Microsoft Endpoint Configuration Manager

File: blueprint/client-devices.md (hunk 10 above)
Comment (noreilly-ccx) [reply blueprint/client-devices.md:1231]: The latest hardening guidance has a VPN as a Required item


image

File: blueprint/abac/intune-configuration.md [hunk 14]
```diff
@@ -349,7 +349,7 @@ The policy GUID in the OMA-URI must be unique to your environment. It can be fou
 
 ### Wi-Fi Configuration
 
-* Name: `WiFi SSID`
+* Name: `W10 WiFi SSID`
```
[the remark this answers is not in the document]
Comment (noreilly-ccx) [reply blueprint/abac/intune-configuration.md:352]: In my repo this is already set as W10 Wifi SSID - I am marking it as resolved

**Comment by zedzedzed** [comment]
Thanks, LGTM

**Pull request merged**